components/style/properties.mako.rs

```diff
@@ -120,10 +122,10 @@ def switch_to_style_struct(name):
     global THIS_STYLE_STRUCT
 
     for style_struct in STYLE_STRUCTS:
-        if style_struct.name == name:
+        if style_struct.trait_name == name:
             THIS_STYLE_STRUCT = style_struct
             return ""
-    fail()
+    raise Exception("Failed to find the struct named " + name)
```

Comment:
`fail()` was apparently an incorrect way to fail in Python, so I changed this to be more helpful while debugging.